com.georgefb.haruna.yml

```diff
@@ -0,0 +1,149 @@
+---
+app-id: com.georgefb.haruna
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: haruna
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--device=dri'
+  - '--filesystem=host'
+  - '--talk-name=ca.desrt.dconf'
```

Comment:
if you need this then you probably need --env=DCONF_USER_CONFIG_DIR=.config/dconf as well, the filesystem permissions should be covered by the host permission.